integration/container/kill_test.go

```diff
@@ -30,35 +30,41 @@ func TestKillContainerInvalidSignal(t *testing.T) {
 }
 
 func TestKillContainer(t *testing.T) {
-	skip.If(t, testEnv.OSType == "windows", "TODO Windows: FIXME. No SIGWINCH")
 	defer setupTest(t)()
 	client := testEnv.APIClient()
 
 	testCases := []struct {
 		doc    string
 		signal string
 		status string
+		skipOs string
 	}{
 		{
 			doc:    "no signal",
 			signal: "",
 			status: "exited",
+			skipOs: "",
 		},
 		{
 			doc:    "non killing signal",
 			signal: "SIGWINCH",
 			status: "running",
+			skipOs: "windows", // Windows does not support SIGWINCH
 		},
 		{
 			doc:    "killing signal",
 			signal: "SIGTERM",
 			status: "exited",
+			skipOs: "",
 		},
 	}
 
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.doc, func(t *testing.T) {
+			if testEnv.OSType == tc.skipOs {
+				return
```

Comment:
guess this could even use `t.Skip()` because it's running as a subtest.

no need to change, unless there's other changes to make

Reply:
I think skipping is ideal here so that the test output shows it as skipped instead of passed.
Can use `skip.If` even like the rest of the tests.

Reply:
Ok. Updated